test/sanity/validate-modules/main.py

```diff
@@ -1082,19 +1084,29 @@ def _validate_docs(self):
 
         return doc_info, doc
 
-    def _check_version_added(self, doc):
-        if not self._is_new_module():
-            return
-
+    def _check_version_added(self, doc, existing_doc):
+        version_added_raw = doc.get('version_added')
         try:
             version_added = StrictVersion(str(doc.get('version_added', '0.0') or '0.0'))
         except ValueError:
             version_added = doc.get('version_added', '0.0')
+            if version_added != 'historical':
```

Comment:
This new conditional allows `version_added == 'historical'` to fall through down to line 1115 where it tracebacks when testing a new module:

```
Traceback (most recent call last):
  File "test/sanity/validate-modules/validate-modules", line 1714, in <module>
    main()
  File "test/sanity/validate-modules/validate-modules", line 1613, in main
    mv.validate()
  File "test/sanity/validate-modules/validate-modules", line 1486, in validate
    doc_info, docs = self._validate_docs()
  File "test/sanity/validate-modules/validate-modules", line 1011, in _validate_docs
    self._check_version_added(doc, existing_doc)
  File "test/sanity/validate-modules/validate-modules", line 1115, in _check_version_added
    if (version_added < strict_ansible_version or
  File "/usr/lib/python3.6/distutils/version.py", line 64, in __gt__
    c = self._cmp(other)
  File "/usr/lib/python3.6/distutils/version.py", line 168, in _cmp
    other = StrictVersion(other)
  File "/usr/lib/python3.6/distutils/version.py", line 40, in __init__
    self.parse(vstring)
  File "/usr/lib/python3.6/distutils/version.py", line 137, in parse
    raise ValueError("invalid version number '%s'" % vstring)
ValueError: invalid version number 'historical'
```

Reply:
Hrm, I thought I had resolved that already.  Maybe I did in one location but not another.

Reply:
Updated